spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'Sign In', type: :feature, js: true do
+  let!(:user) { FactoryGirl.create( :user, password: '123456' ) }
+
+  before (:each) do
+    visit root_path
+  end
+
+  scenario 'Sucessful sign in' do
+    user.confirm
```

Comment:
You're doing the same action in your scenarios, for example `user.confirm` `click_button  "Sign in"` etc.

Try to put these actions inside a `before (:each)` block when its possible